### DISCUSSION TITLE
谁可不可以给我演示一下如何使用，我在瓦中测试的时候并不是很会用

### DISCUSSION THREAD

**Discussion by 1fonlyU** [Passer1072/RookieAI_yolov8#51] [General]
<img width="1113" alt="image" src="https://github.com/user-attachments/assets/ba1536d3-b536-4a9a-ab34-65e477a0841f">

**Comment by 1fonlyU** [comment]
<img width="1190" alt="image" src="https://github.com/user-attachments/assets/a967607b-490d-4f6c-9c7f-00192ebec214">
使用这个按键的触发方式的时候鼠标也不会移动，用的移动库也是罗技的不是win32的，为什么呢

**Reply by Passer1072** [reply]
> <img width="1190" alt="image" src="https://github.com/user-attachments/assets/a967607b-490d-4f6c-9c7f-00192ebec214">
> 
> 使用这个按键的触发方式的时候鼠标也不会移动，用的移动库也是罗技的不是win32的，为什么呢

国际服由于反作弊问题进入游戏后会屏蔽掉其他鼠标移动方式，仅允许开始游戏时使用的鼠标进行移动，目前猜测只有kmbox从硬件层面绕过才能使用。国服不清楚。
